content-src/components/DiscoveryStreamBase/DiscoveryStreamBase.jsx

```diff
@@ -189,6 +189,8 @@ export class _DiscoveryStreamBase extends React.PureComponent {
     }
 
     // Find the first component of a type and remove it from layout
+    // Note: Do not use the return component for additional FluentDOM use. Otherwise,
+    // the fluent IDs will be stripped and replaced with copy/replace strings.
```

Comment:
"How it works" wasn't loading in Discovery Stream layout, but was working in regular view. Looks like its because we load Top Sites and Recommended by Pocket section twice (one for regular and one for DS), but choose to extract the first instance and copy over the properties from the already inserted DOM node.

Reply:
This wasn't working because the strings from layout are translated not ids. The DOM nodes are not already inserted -- the extraction happens from the layout definition. If you inspect the link, you'll see it tried to do:

```html
<a href="https://getpocket.com/firefox/new_tab_learn_more">
  <span data-l10n-id="How it works"></span>
</a>```

Reply:
I didn't properly read what was going on here. Thanks for clarifying here and in the other comment. I tried to revert the changes in the next commit (a bit of a hacky fix though).